memebot/command.py

```diff
@@ -7,12 +7,12 @@
 log = logging.getLogger('memebot')
 
 
-ExecutorCallable = Callable[[str, discord.abc.Messageable], Awaitable[None]]
+ExecutorCallable = Callable[['Command', discord.abc.Messageable], Awaitable[None]]
```

Comment:
Just wondering if `Command` is a forward reference to type that hasn't been defined here or if it's just an importable type that you could use to type hint here and are just choosing not to?

Reply:
It's a forward type reference